index.html

```diff
@@ -1,9 +1,15 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="bg-background">
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
hmm, technically it's correct to use cdn servers to reduce site loading time, but be wary that this is an external dependency and if it will fail then we might have problems loading our frontend. I prefer hosting fonts in our project assets and load them from there.

I wonder if it is possible to have a hybrid approach - the same font from both cdn and locally? Haven't tried it yet.

Reply:
This is just information - you don't have to change anything here.